### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by 184446223** [facebookresearch/omnivore#14]
Hello author, I am very interested in your code!
I would like to ask if it is possible to publish the code of the loss function for the training period, and I would like to know if the labels of the 3 datasets are constrained separately or together. Or is it computing 3 classification tasks or one classification task.

**Comment by mannatsingh** [comment]
Closing this out, please feel free to re-open if your question isn't addressed
